.github/scripts/crib/pr-comment-crib-env.js

```diff
@@ -19,9 +34,19 @@ async function commentExists(octokit, owner, repo, prNumber, uniqueIdentifier) {
 async function run() {
   try {
     const token = process.env.GITHUB_TOKEN;
-    const octokit = new Octokit({ auth: token });
+    const route53ZoneId = process.env.ROUTE53_ZONE_ID;
+
+    // Check for the existence of GITHUB_TOKEN and ROUTE53_ZONE_ID
+    if (!token || !route53ZoneId) {
```

Comment:
```suggestion
```
Redundant check

Reply:
dffc5914d477a8afc2f5c8d060df203e83eef8e9

Reply:
ah, i meant you should remove the `||` outer `if` statement, having two individual ones would be better since it provides a more specific reason into which env var is missing